lib/workos.ex

```diff
@@ -1,8 +1,10 @@
 defmodule WorkOS do
   @moduledoc """
-  Use the WorkOS module to authenticate your requests to the WorkOS API
+  Documentation for `WorkOS`.
   """
 
+  @config_module WorkOS.Client
```

Comment:
`@config_module` is a module attribute that is used to specify the module where configuration settings are stored. 

I also updated the `@moduledoc` here for when we publish to [hexdocs](https://hexdocs.pm)